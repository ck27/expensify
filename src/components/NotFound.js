import React from "react";

const NotFound = () => (
    <div>
        <h1>Looks like you took a wrong turn :(</h1>
        <Link to="/">Click to go HOME</Link>
    </div>
);

export default NotFound;