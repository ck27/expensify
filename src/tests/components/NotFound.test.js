import React from "react";
import {shallow} from "enzyme";
import NotFound from "../../components/NotFound";

test("it should render NotFound", () => {
    const content = shallow(<NotFound />);
    expect(content).toMatchSnapshot();
})