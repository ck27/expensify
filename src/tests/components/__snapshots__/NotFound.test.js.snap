// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`it should render NotFound 1`] = `
<div>
  <h1>
    Looks like you took a wrong turn :(
  </h1>
  <Link
    to="/"
  >
    Click to go HOME
  </Link>
</div>
`;
